sql/core/src/test/resources/sql-tests/inputs/postgreSQL/interval.sql

```diff
@@ -0,0 +1,330 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- INTERVAL
+-- https://github.com/postgres/postgres/blob/REL_12_STABLE/src/test/regress/sql/interval.sql
+
+-- SET DATESTYLE = 'ISO';
+-- SET IntervalStyle to postgres;
```

Comment:
This one is natural.

Reply:
Created this ticket: https://issues.apache.org/jira/browse/SPARK-29406